app/components/work_packages/activities_tab/journals/item_component/edit.sass

```diff
@@ -0,0 +1,13 @@
+.work-packages-activities-tab-journals-item-component-edit
+    // prototypical primer style adoptions
+    .ck-toolbar
+        border-radius: var(--borderRadius-medium) var(--borderRadius-medium) 0px 0px!important
+    .document-editor__editable-container
+        background: var(--color-scale-white)
+        border-radius: 0px 0px var(--borderRadius-medium) var(--borderRadius-medium)!important
+    .ck-content
+        background: var(--color-scale-white)
+        overflow-y: auto
+        border-radius: var(--borderRadius-medium)!important
+        margin: 5px
```

Comment:
I'd prefer we place them somewhere like "ckeditor_primer_overrides" or `primer/_overrides` and pass a class to CKEditor so it becomes reusable. (Note to self: check with primer)

Reply:
Good point! Done, it's now reusable!